docs/hazmat/primitives/key-derivation-functions.rst

```diff
@@ -506,6 +506,93 @@ Different KDFs are suitable for different tasks such as:
         ``key_material`` generates the same key as the ``expected_key``, and
         raises an exception if they do not match.
 
+.. currentmodule:: cryptography.hazmat.primitives.kdf.x963kdf
+
+.. class:: X963KDF(algorithm, length, otherinfo, backend)
+
+    .. versionadded:: 1.0
+
+    X963KDF (ANSI X9.63 Key Derivation Function) is defined by ANSI
+    X9.63-2001 `ANSI X9.63:2001`_ document, to be used to
+    derive keys for use after a Key Exchange negotiation operation.
```

Comment:
On Fri, 2015-09-18 at 17:23 -0700, Alex Gaynor wrote:
> Should we mention that this is derpcated by ANSI and should only be
> used by existing applications?

I do not have access to X9.63:2011 where is it said this is deprecated ?